lametro/models.py

```diff
@@ -664,6 +664,14 @@ def ecomment_message(self):
         else:
             return self.UPCOMING_ECOMMENT_MESSAGE
 
+    @property
+    def todays_meetings(cls):
+        now = app_timezone.localize(datetime.now())
+
+        end_of_day = app.timezone.localize(datetime(now.year, now.month, now.day, 17, 0))
+
+        return cls.objects.filter(start_time__gte=now, start_time__lte=end_of_day)
```

Comment:
Per the issue description, the ultimate goal is to "show the day's meetings and their status (passed, live, scheduled or something to that effect)". Will this query return all meetings on a given date?

Reply:
ah, thanks for bringing my attention back to that part of the issue description. This would remove meetings from the list if they have already started (if they are live or passed).